user_guide_src/source/testing/benchmark.rst

```diff
@@ -38,6 +38,20 @@ and stop timers:
 
 .. literalinclude:: benchmark/003.php
 
+If you use very small code blocks to benchmark, you can also use the ``record()`` method. It accepts
```

Comment:
Including the added version would reduce misunderstandings by users of the older version.

```suggestion
Since v4.3.0, if you use very small code blocks to benchmark, you can also use the ``record()`` method. It accepts
```